app/templates/releases/index.hbs

```diff
@@ -128,3 +128,36 @@
   MINOR versions may introduce new features so long as they are backwards compatible,
   and PATCH releases may include bug or security fixes.
 </p>
+
+<h3>Ecosystem tools</h3>
+<p>
+  There are additional considerations for our support policies for other major
+  ecosystem tools, specifically <a href="https://nodejs.org/en/">Node.js</a> and
+  <a href="https://www.typescriptlang.org">TypeScript</a>.
+</p>
+
+<h4>Node.js</h4>
+<p>
+  Ember supports current stable and <abbr title="long-term support">LTS</abbr>
+  releases for Node.js. When a version of Node.js goes out of its long-term
+  support window, Ember also drops support for it. <em>This is not considered a
+  breaking change.</em> However, release notes will include
```

Comment:
```suggestion
  breaking change.</em> However, the release notes will mention this.
```

Reply:
Ah, good catch, thanks! I'm going to elaborate it even further:

```suggestion
  breaking change.</em> However, release notes will always include a notice
  when a previously supported Node version becomes unsupported because it left
  its <abbr title="long-term support">LTS</abbr> period.
```